pymc/gp/hsgp_approx.py

```diff
@@ -252,8 +317,8 @@ def prior_linearized(self, Xs: TensorLike):
 
         Correct results when using `prior_linearized` in tandem with `pm.set_data` and
         `pm.Data` require two conditions.  First, one must specify `L` instead of `c` when
-        the GP is constructed.  If not, a RuntimeError is raised.  Second, the `Xs` needs to be
-        zero-centered, so its mean must be subtracted.  An example is given below.
+        the GP is constructed.  If not, a RuntimeError is raised.  Second, the `Xs` needs to be  ### CHECH WITH c INSTEAD!!
```

Comment:
Do you want to remove `### CHECH WITH c INSTEAD!!` ?

Reply:
(same below)

Reply:
Good catch